features/gpc.yml

```diff
@@ -0,0 +1,7 @@
+name: Global Privacy Control
```

Comment:
Probably just need the linter to automatically lowercase non-first-words here and call it done